File: clients/javascript/src/models/operationBehavior.ts
// this file is @generated

export enum OperationBehavior {
    Upsert = 'upsert',
    Insert = 'insert',
    Update = 'update',
    }

export const OperationBehaviorSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): OperationBehavior {
        return object;
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: OperationBehavior): any {
        return self;
    }
}

File: z-clients/java/src/main/java/com/svix/diom/models/OperationBehavior.java
// this file is @generated
package com.svix.diom.models;

import com.fasterxml.jackson.annotation.JsonValue;

public enum OperationBehavior {
    UPSERT("upsert"),
    INSERT("insert"),
    UPDATE("update");
    private final String value;

    OperationBehavior(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }
}